data/settings.js

```diff
@@ -11010,8 +11012,24 @@ You may need to recompile OpenLDAP with debugging enabled to get enough output.`
 Specify dereference which is set as an LDAP option.`
 	},
 
+	ldap_filter: {
```

Comment:
It's now called dict_map_ldap_filter

Reply:
Or actually it will be in `DOV-7288` which has its own documentation change also.

Reply:
Although that documentation is missing dict_map_ldap_filter, so maybe easiest to just rename it here already. :)

Reply:
.. DOV-7288 is now merged, so need to fix it here in any case.